evm/utils/keccak.py

```diff
@@ -10,4 +10,9 @@ def keccak(value):
     return keccak_256(value).digest()
 
 
+# FIXME: this is a terrible name.
+def keccak_updateable(value):
```

Comment:
why?

Reply:
I don't know, it just sounds like a weird name

Reply:
No, I mean why does it exist?

Reply:
Because the existing one give us the .digest(), but for the egress/ingress macs (created in derive_secrets) we need a [hasher which allows us to continuously update its value before getting the digest](https://github.com/pipermerriam/py-evm/pull/98/files#diff-5dad8f91b2b80753e0030b15f47c3931R111)

Reply:
Got it.  I was somehow failing to map the term updatable to the hash.  I was thinking you were namespacing a mirrored version of keccak to be *updated* with some other function at a later time.  👍